src/Switcher/Switcher.tsx

```diff
@@ -20,6 +20,12 @@ const Switcher: React.FC<SwitcherProps> = ({ selected, onChange, ...rest }) => {
     return value === selected;
   };
 
+  const getSelectedIndex = () => {
+    const options = rest.children.map((child) => child.props.value);
+
+    return options.indexOf(selected);
```

Comment:
Should we use `findIndex` instead of `map` so that we don't loop through all elements and instead break as soon as the first child with the condition is found?